oxcaml/tests/backend/llvmize/dune.inc

```diff
@@ -0,0 +1,45 @@
+
+(rule
+ (enabled_if (and (= %{context_name} "main") (= %{architecture} "amd64")))
+ (targets id_fn.output.corrected)
+ (deps (:ml id_fn.ml) filter.sh)
+ (action
+  (with-outputs-to
+   id_fn.output.corrected
+   (pipe-outputs
+    (run
+     %{bin:ocamlopt.opt} %{ml} -c -llvm-backend -stop-after llvmize -keep-llvmir)
+    (run cat id_fn.ll)
+    (run ./filter.sh)))))
+
+(rule
+ (enabled_if (and (= %{context_name} "main") (= %{architecture} "amd64")))
+ (alias runtest)
+ (deps id_fn.output id_fn.output.corrected)
+ (action
+  (diff id_fn.output id_fn.output.corrected)))
+
+(rule
+ (enabled_if (and (= %{context_name} "main") (= %{architecture} "amd64")))
+ (targets const_val.output.exe)
+ (deps (:ml const_val_bootstrap.ml const_val.ml))
+ (action
+  (pipe-outputs
```

Comment:
what does `pipe-outputs` do here? should it be `progn` instead to run a sequence of commands?

Reply:
this may be the cause of ci test failures.